### PR TITLE
Tic tac toe

### PR DESCRIPTION
**Pull request by listenToRipley**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] All tests passed: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript: 30 points

**Checkpoint 2**
- [ ] The application works as it should: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript and the DOM API: 30 points

**Checkpoint 3**
- [ ] Use of React: 25 points
- [ ] Accesses an API: 25 points
- [ ] Proper use of documentation (commenting on code): 25 points
- [ ] The application functions as it should: 25 points

### CODE REVIEW

**Comment by listenToRipley** [comment]
Line 68 - 77 should have been deleted

**Comment by listenToRipley** [comment]
made this on the wrong branch

File: 02week/todo.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+for(let i=0; i<allDeletes.length; i++) {
+  let deleteButton = allDeletes[i]
+  setupDeleteEvent(deleteButton);
+}
+
+function setupDeleteEvent(deleteButton) {
+  deleteButton.addEventListener('click', function() {
+    console.log('Delete got clicked! parent li is, ', deleteButton.parentElement)
+    let parentLi = deleteButton.parentElement
+    parentLi,remove()
```
Comment (yousifS) [inline 02week/todo.js:59]: should be `parentLi.remove()`

File: 02week/todo.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
+body {
```
Comment (yousifS) [inline 02week/todo.css:1]: nice!

File: 02week/ticTacToe.js [hunk 3]
```diff
@@ -24,6 +24,48 @@ function printBoard() {
...
+
+//this is a helper function for checkForWin()
+//return true if any row wins
+function horizontalWin() {
+  // Your code here
+  //loops through each option for the second array
+  for (let i = 0; i >= 2; i++) {
+
+    //checked is any of the whole rows returns with the same value 
+    if (((board[0][i]) || (board[1][i]) || (board[2][i])) === playerTurn)
```
Comment (yousifS) [inline 02week/ticTacToe.js:37]: this does not check for all 3 cells being equals. you would have to check each pair separately
```
if(board[0][i] == playerTurn && board[1][i] == playerTurn && board[2][i] == playerTurn)

```

**Pull request merged**